components/grid/style/mixin.less

```diff
@@ -29,27 +29,6 @@
   .col(1);
 }
 
-.float-grid-columns(@class) {
-  .col(@index) {
-    // initial
-    @item: ~'.@{ant-prefix}-col@{class}-@{index}';
-    .col((@index + 1), @item);
-  }
-  .col(@index, @list) when (@index =< @grid-columns) {
-    // general
-    @item: ~'.@{ant-prefix}-col@{class}-@{index}';
-    .col((@index + 1), ~'@{list}, @{item}');
-  }
-  .col(@index, @list) when (@index > @grid-columns) {
-    // terminal
-    @{list} {
-      flex: 0 0 auto;
-      float: left;
-    }
-  }
-  .col(1); // kickstart it
-}
-
 .loop-grid-columns(@index, @class) when (@index > 0) {
```

Comment:
Can we remove all generated loop code？

Reply:
I will try it later

Reply:
`Grid` base on 24 range, so `.loop-grid-column` class is important, but I change it with `flex`, get a better experience